lib/auth_web/controllers/app_controller.ex

```diff
@@ -35,7 +35,7 @@ defmodule AuthWeb.AppController do
     app = App.get_app!(id)
     #  restrict viewership to owner||admin https://github.com/dwyl/auth/issues/99
     if conn.assigns.person.id == app.person_id || conn.assigns.person.id == 1 do
-      render(conn, "show.html", app: app)
+      render(conn, "show.html", app: app, auth_url: @auth_url)
```

Comment:
We could just use `AuthPlug.Helpers.get_baseurl_from_conn/1` for the `auth_url`. 💭

Reply:
I've looked at the `get_baseurl_from_conn` function:
```elixir
  def get_baseurl_from_conn(%{host: h, port: p}) when h == "localhost" do
    "http://#{h}:#{p}"
  end

  def get_baseurl_from_conn(%{host: h}) do
    "https://#{h}"
  end
```
It adds the https:// or http:// to the host and because we do not want this part to the api key I think we can just use `conn.host` directly.

Reply:
I've done the change to use conn.host, however the mix ecto.reset is failing when the mix environment is defined as `test` but working with `dev`, so the tests are currently failing. I think this is due to how the .env file is sourced for the tests, still trying to find the code line of the error

Reply:
The issue was due to the dependencies not updated for tests environment. I've deleted the `deps` folder to force the app to get and compile them again.
The application is working and all tests are passing. I'm cleaning all the `IO.inspect` I've added while debugging and I'll add the new commit

Reply:
@SimonLab makes sense. Thanks. 👍